components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -480,46 +481,48 @@ void CosmeticFiltersJSHandler::ApplyRules(bool de_amp_enabled) {
 
   CSSRulesRoutine(*resources_dict_);
 
-  bool has_removals = false;
-  //: remove()
-  std::string remove_selectors_json;
-  const auto* remove_selectors_list =
-      resources_dict_->FindList("remove_selectors");
-  if (remove_selectors_list && !remove_selectors_list->empty()) {
-    base::JSONWriter::Write(*remove_selectors_list, &remove_selectors_json);
-    has_removals = true;
+  bool has_procedural_actions = false;
```

Comment:
For other things in this file we had previously talked about moving the logic to the browser side and filtering there instead of sending all the resources and filtering on the renderer. As we increase the number of things we're sending we might start to run into mojo size limits, but also it's just generally better to consolidate this logic on the browser side and send less data.